src/main/java/jupyter/Displayers.java

```diff
@@ -57,8 +57,10 @@ public static void setMimeTypes(String... types) {
    */
   @SuppressWarnings("unchecked")
   public static <T> Map<String, String> display(T obj) {
-    return registration()
-        .find((Class<T>) obj.getClass())
-        .display(obj);
+    Displayer<? super T> displayer = registration().find((Class<T>) obj.getClass());
+    if (displayer != null)
+      return displayer.display(obj);
+    else
+      return null;
```

Comment:
This changes the contract of display. I think it should return `Collections.emptyMap` instead so that it doesn't break callers than rely on this always returning a non-null map.

Reply:
I guess I was assuming that the change was to `find` instead of to `display`. Would changing `find` work for you instead? Alternatively, would using `emptyMap` and checking whether it is empty work?

Reply:
Settled on returning `emptyMap` in `display` (it's even for me - I'll keep using `find` from almond I think).